Sources/Sentry/SentryNetworkTrackingIntegration.m

```diff
@@ -1,22 +1,31 @@
 #import "SentryNetworkTrackingIntegration.h"
+#import "SentryLog.h"
 #import "SentryNetworkSwizzling.h"
 #import "SentryNetworkTracker.h"
 #import "SentryOptions.h"
 
-@interface
-SentryNetworkTrackingIntegration ()
-
-@property (nonatomic, strong) SentryOptions *options;
-@end
-
 @implementation SentryNetworkTrackingIntegration
 
 - (void)installWithOptions:(SentryOptions *)options
 {
-    self.options = options;
-    if (options.enableAutoPerformanceTracking) {
-        [SentryNetworkSwizzling start];
+    // We don't check isTracingEnabled, because the integration
+    // also creates breadcrumbs for HTTP requests.
+    if (!options.enableAutoPerformanceTracking) {
```

Comment:
l: And in the case that I want only to track network request but not UI performance?
I would prefer this two flags to be independents.

Reply:
Good point. This should actually be `h` 😁. Will change it.

Reply:
I guess you still want HTTP breadcrumbs when you disable `enableAutoPerformanceTracking`. I'm going to change it. This should be `h` 😁

Reply:
I changed my mind. I'm not going to fix this right now. I put a comment into this class

> We are aware that the SDK only creates breadcrumbs for HTTP requests if performance is
> enabled. A proper fix is not straightforward as we need several checks on SentryOptions in
> SentryNetworkTracker. As we have a problem with KVO, see
> https://github.com/getsentry/sentry-cocoa/issues/1328, we don't know if we can keep the
> SentryNetworkTracker (written on 29th of September 2021). Therefore we accept this tradeof
> for now.